libnmstate/nm/wired.py

```diff
@@ -141,6 +143,17 @@ def get_info(device):
     return info
 
 
+def _get_mac_address_from_sysfs(ifname):
```

Comment:
IMHO there should be a bug for NetworkManager to always provide the MAC and the bug should be referenced here in a docstring.

Reply:
Done